content/md/first-steps/manage-your-attributes.md

```diff
@@ -48,6 +48,8 @@ Please note that the **following attribute codes cannot be used** in Akeneo PIM:
 ## Add attributes validation parameters
 
 Each attribute has specific properties to add more control on attribute's values:
+- **Yes/No attribute**:
+  - *Default value*: the value that is put in each Yes/No attribute when creating a new product/product model
```

Comment:
It's not a validation parameter, but rather an attribute type specific parameter (like asset family for asset collection, measurement family for measurements...)

Reply:
I found weird "the value that is put (or added) in each Yes/No attribute"  

I suggest somethinkg like: 

"the value used at product/product model creation when no value is specified for this attribute"